pkg/api-server/oapi-helpers/helpers.go

```diff
@@ -12,9 +12,10 @@ func ResourceToMeta(m core_model.Resource) api_common.Meta {
 
 func ResourceMetaToMeta(resType core_model.ResourceType, m core_model.ResourceMeta) api_common.Meta {
 	return api_common.Meta{
-		Type: string(resType),
-		Mesh: m.GetMesh(),
-		Name: m.GetName(),
+		Type:   string(resType),
+		Mesh:   m.GetMesh(),
+		Name:   m.GetName(),
+		Labels: m.GetLabels(),
```

Comment:
@johncowen  can confirm but I think we agreed on empty objects instead of nil when entries is empty

Reply:
Got it.